src/applications/simple-forms/21-0845/definitions/form-definitions.js

```diff
@@ -0,0 +1,31 @@
+// definitions with form-specific overrides
+export default {
+  pdfEmail: {
+    type: 'string',
+    maxLength: 40, // PDF-specific override
+    format: 'email',
+  },
+  pdfFullNameNoSuffix: {
+    // has PDF-specific maxLengths
+    type: 'object',
+    additionalProperties: false,
+    required: ['first', 'last'],
+    properties: {
+      first: {
+        type: 'string',
+        minLength: 1,
+        maxLength: 12, // PDF-specific override
```

Comment:
Let's ask Jeana/Andrew about this approach.

Reply:
Jeana just said we should set PDF-specific maxLengths going fwd.